### PR TITLE
Use new hub interface to download op

### PR DESCRIPTION
**Pull request by junjiejiangjjj**

### CODE REVIEW

File: towhee/hub/downloader.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                if os.path.exists(f.name):
+                    os.remove(f.name)
+
+    def download(self):
+        futures = []
+        with ThreadPoolExecutor(max_workers=5) as pool:
+            for local_file, url in self._hub_files.local_url_pair():
+                if not local_file.is_file():
+                    futures.append(pool.submit(_Downloader.download_url_to_file, url, local_file))
+            _ = [i.result() for i in futures]
```
Comment (xiaofan-luan) [inline towhee/hub/downloader.py:158]: exception handling?

**Comment by xiaofan-luan** [comment]
overall lgtm

File: towhee/hub/downloader.py (hunk 1 above)
Comment (junjiejiangjjj) [reply towhee/hub/downloader.py:158]: This function would raise the exception, and the `operator_loader` will catch the exception, print logs and stop running.

**Pull request merged**